src/GHC.hs

```diff
@@ -29,7 +29,7 @@ defaultKnownPackages =
     , filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal, ghci
     , ghcPkg, ghcPrim, ghcTags, haddock, haskeline, hsc2hs, hoopl, hp2ps, hpc
     , hpcBin, integerGmp, integerSimple, iservBin, libffi, mkUserGuidePart
-    , parallel, pretty, primitive, process, rts, runGhc, stm, templateHaskell
+    , pretty, process, rts, runGhc, templateHaskell
```

Comment:
OK, I will give it a try. I would like to rebase this PR to your latest master. Do you have any objections?